src/content/app/genome-browser/components/browser-track-legend/BrowserTrackLegend.tsx

```diff
@@ -59,8 +59,13 @@ const BrowserTrackLegend = (props: Props) => {
     if (!isTrackLegendHotspot(payload)) {
       return;
     }
+
+    const { bottom, top, right } = payload['hotspot-area'];
+    const toolTipX = right;
```

Comment:
Isn't it a bit too far? Here's what it looks like:

![image](https://user-images.githubusercontent.com/6834224/225594849-dcab62bb-f5cd-4808-923e-4cab23470a55.png)

![image](https://user-images.githubusercontent.com/6834224/225594996-ec97ab29-55d7-4a05-85aa-f761d53ad61b.png)

Compare with Andrea's directions in the ticket:

![image](https://user-images.githubusercontent.com/6834224/225595145-44b14810-0c58-4d31-80b3-21bac4821deb.png)

![image](https://user-images.githubusercontent.com/6834224/225595203-3bb77115-3b2c-412c-afdf-c9979b32aa44.png)